wasm/bls-utils/benches/wasm_verify.rs

```diff
@@ -0,0 +1,57 @@
+#[path = "../src/spec/mod.rs"] mod spec;
+
+use spec::{bindings::Runtime, types::VerifyParams};
+use std::{fs::read, path::Path};
+use bls_signatures::Serialize;
+use group::GroupEncoding;
+use bls_utils::make_sig_safe;
+use criterion::{criterion_group, criterion_main, Criterion};
+
+
+
+macro_rules! bench_verify {
+    ($name:ident, $num:expr) => {
+
+        fn $name(c: &mut Criterion) {
+            c.bench_function(&format!("wasm_verify {}", $num), |b| {
+                let wasm_file = Path::new("./src/wasm-files/bls_utils.wasm");
+                let bytes = read(wasm_file).unwrap();
+                let mut runtime = Runtime::new(&bytes).unwrap();
+            
+                let (aggregated_signature, hashes, public_keys, _) = make_sig_safe($num, 64);
+    
+                let signature_bytes = aggregated_signature.as_bytes().to_vec();
+                let public_keys: Vec<_> = public_keys.iter().map(|x| x.as_bytes()).collect();
+                let hash_vec: Vec<_> = hashes
+                    .iter()
+                    .map(|x| x.to_bytes().as_mut().to_vec())
+                    .collect();
+            
+                let params = VerifyParams {
+                    aggregate_signature: signature_bytes,
+                    pub_keys: public_keys,
+                    hashes: hash_vec,
+                };
+
+                b.iter(|| runtime.run_sig_verification(params.clone()))
```

Comment:
Not a priority but is there a simple way to also measure the Gas of the call at this point and log it / record it in criterion ?

Reply:
No straightforward way to track gas.

I've added a global wasm var called "gas_counter" that gets imported from the runtime when running which how I believe 'fvm-wasm-instrument' intended to use that variable. There is however no straightforward way that I found to read that value from wasm after it has been modified as it is considered an import from the host. I can figure this out though, but if not a priority we can skip for this PR